takhojin/routes/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+router.use("/users", userRouter.router);
+// /users -> 이 경로가 입력되면 그 옆을 실행
+const postRouter = require("./postRouter");
+router.use("/posts", postRouter.router);
+
+module.exports = router;
```

Comment:
```suggestion
const express = require("express");

const userRouter = require("./userRouter");
const postRouter = require("./postRouter");

const router = express.Router();

router.use("/users", userRouter.router);
router.use("/posts", postRouter.router);

module.exports = router;
```
주석은 지워주시고, 필요한 모듈들을 불러오는 코드가 가장 위에 작성될 수 있도록 해주세요!